src/Stryker.Core/Stryker.Core/Compiling/RollbackProcess.cs

```diff
@@ -123,25 +125,30 @@ public RollbackProcessResult Start(CSharpCompilation compiler, ImmutableArray<Di
             return (null, -1);
         }
 
-        private int? ExtractMutationIfAndId(SyntaxNode node)
+        private (int? Id, string Engine, string Type) ExtractMutationInfo(SyntaxNode node)
```

Comment:
Since we use this construct in multiple places, and it is not only used to simply return multiple values, I think we should create a private class for it.

Reply:
This helps avoid having to do this with 'Item3' https://github.com/stryker-mutator/stryker-net/pull/1875/files#diff-6a4265e13c259acff0fccc6bb044650b26193ed6c95b6cd6b88c99ff2ba401bdR238